### DISCUSSION TITLE
Outdated version of GitHub project loading in Android Studio

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22744] [New to GitHub]
<p>Hi,</p>
<p>I’m trying to use a library from GitHub in my Android Studio Project (specifically, the Universal Image Loader <a href="https://github.com/nostra13/Android-Universal-Image-Loader" rel="nofollow noopener">https://github.com/nostra13/Android-Universal-Image-Loader</a>).</p>
<p>My project has been crashing, due to an error with a variable called  <em>mMaxHeight</em> , in a file called  <strong>ImageViewAware.java</strong></p>
<p>On the GitHub page, I see that this issue was 4 fixed months ago: <a href="https://i.stack.imgur.com/lGs85.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/64666557/181072780-f41620cc-42c9-4c6e-a543-dd021dd3e586.png" alt="enter image description here" data-base62-sha1="ua3xpajb8LTqlH0SK4lBD9YEs0J" width="690" height="257"></a></p>
<p>When I view the code in  <strong>ImageViewAware.java</strong>  on GitHub, I see that the issue is fixed.</p>
<p>However when I look at the version of  <strong>ImageViewAware.java</strong>  being used in my Android Studio, I see it’s an older version, with the issue not fixed.</p>
<p>I’ve added</p>
<pre><code class="lang-auto">implementation 'com.nostra13.universalimageloader:universal-image-loader:1.9.5'
</code></pre>
<p>to my Gradle Dependency, as the documentation says, and have followed all other instructions I can see.</p>
<p>I notice that the last update to those instructions was over a year ago. Should I be implementing a newer version than 1.9.5? Is there another reason why the version in my Android Studio isn’t getting the latest GitHub version?</p>
<p>I’m pretty new to this and feeling a bit out of my depth. Thanks for any help!</p>
<p>Alex</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/chalni">@chalni</a>! :wave: Welcome to the Community!</p>
<p>Your best bet to get help with that project is to talk to the people who built it. There are a few places we suggest you look when you need help with a project hosted on GitHub:</p>
<ul>
<li>its SUPPORT or README file, which may include contact information or an official website</li>
<li>its Issues tab, where you can report bugs</li>
<li>the owner’s GitHub profile, which may include contact information</li>
</ul>
<p>Of course, do let us know if you have any questions about GitHub itself!</p>
